packages/templates/introspection/src/config.ts

```diff
@@ -0,0 +1,22 @@
+import { EInputType, GeneratorConfig } from 'graphql-codegen-core';
+import * as index from './template.handlebars';
+import { toJSON } from './helpers/to-json';
+
+export const config: GeneratorConfig = {
+  inputType: EInputType.SINGLE_FILE,
+  flattenTypes: true,
```

Comment:
I don't know if I should skip that or leave it as is

Reply:
Just leave it as it